src/olympia/translations/fields.py

```diff
@@ -57,7 +157,7 @@ def db_column(self, value):
         if value is not None:
             self._db_column = value
 
-    def contribute_to_class(self, cls, name):
+    def contribute_to_class(self, cls, name, private_only=False, **kwargs):
         """Add this Translation to ``cls._meta.translated_fields``."""
         super(TranslatedField, self).contribute_to_class(cls, name)
```

Comment:
should `private_only` be passed down to the `super(...).contribute_to_class(...)` ?

Reply:
Indeed, I fixed this in eab651223c